pkg/apis/hive/v1alpha1/dnszone_types.go

```diff
@@ -32,6 +32,12 @@ type DNSZoneSpec struct {
 	// Zone is the DNS zone to host
 	Zone string `json:"zone"`
 
+	// LinkToParentDomain specifies whether DNS records should
+	// be automatically created to link this DNSZone with a
+	// parent domain.
+	// +optional
+	LinkToParentDomain bool `json:"linkToParentDomain,omitempty"`
```

Comment:
Sorry for the very basic question but why do we still have a DNSZone CRD in light of the DNSEndpoint, I don't fully understand the differences.

Reply:
DNSZone represents a route53 hosted zone. A DNSEndpoint is simply a collection of route53 records. Unfortunately, right now it cannot be named anything other than `DNSEndpoint`. This is something that should be fixed upstream. My preference would be to call this a DNSRecordSet